src/main/java/org/symphonyoss/symphony/messageml/elements/Element.java

```diff
@@ -449,6 +450,20 @@ void assertContainsChildOfType(Collection<Class<? extends Element>> elementTypes
     }
   }
 
+  /**
+   * Assert that children with any of the informed types do not exceed the maximum count allowed.
+   *
+   * @param elementTypes The element types that will be verified in order to ensure that the maximum count was not exceed.
+   * @throws InvalidInputException
+   */
+  void assertChildrenNotExceedingMaxCount(Collection<Class<? extends Element>> elementTypes) throws InvalidInputException {
+    boolean hasExceeded = elementTypes.stream().anyMatch(type -> findElements(type).size() > MAX_COUNT_PER_CHILD_TYPE);
```

Comment:
The "MAX_COUNT_PER_CHILD_TYPE" should be a parameter, not fixed in here. Then on the Form class you have the 20